### PR TITLE
Add language samples

### PR DESCRIPTION
**Pull request by anupama-pathirage**
- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/main-pages/ballerina.io-website-theme/samples/api-constructs.bal [hunk 1]
```diff
@@ -13,15 +11,15 @@ service<http:Service> hello bind {port: 9090} {
...
     hi (endpoint caller, http:Request request) {
-        // Create empty response struct
+        // Create empty response
         http:Response res;
         // Try to retrieve parameters
         var  payload = request.getTextPayload();
         // Different handling depending on if we got proper string or error
         match payload {
-            string name => {res.setTextPayload("Hello " + untaint name + "!\n");}
+            string n => {res.setTextPayload("Hello " + untaint n + "!\n");}
```
Comment (afkham) [inline website/main-pages/ballerina.io-website-theme/samples/api-constructs.bal]: Why did you change name to n?

File: website/main-pages/ballerina.io-website-theme/samples/asynchronous.bal [hunk 2]
```diff
@@ -11,22 +11,7 @@ function main(string... args) {
 
 function sum (int a, int b) returns (int) { return a + b; }
 
-// functions can spawn multiple parallel worker threads
-// that can fork, join, and exchange data
 function square_plus_cube(future<int> f) returns int {
```
Comment (afkham) [inline website/main-pages/ballerina.io-website-theme/samples/asynchronous.bal]: function names should be camel case.

Comment (anupama-pathirage) [reply website/main-pages/ballerina.io-website-theme/samples/asynchronous.bal]: Fixed with https://github.com/ballerina-platform/ballerina-www/pull/1141/commits/08400527cd926318db39de841c662e72f4961ee4

File: website/main-pages/ballerina.io-website-theme/samples/api-constructs.bal (hunk 1 above)
Comment (anupama-pathirage) [reply website/main-pages/ballerina.io-website-theme/samples/api-constructs.bal]: It was actually done to reduce the line width to avoid horizontal scroll bar. Anyway changed back to name and did formatting.
